Root/Backend/src/controller/stock-price.js

```diff
@@ -0,0 +1,27 @@
+import dotenv from "dotenv";
+import "isomorphic-fetch"
+dotenv.config({ silent: process.env.NODE_ENV === 'production' });
+
+
+const fetchCaller = (url, res, stock) => {
+	fetch(url)
+	  	.then(response => response.json())
+	  	.then(data => {
+	  		const price = data["Global Quote"]["05. price"]
```

Comment:
What is 05. price?

Reply:
Yeah, I'm hard coding here. I think 05. stands for the 5 value in the json. The results are standard for this API but for some reason they have this weird naming.